lenovo/thinkpad/x1/9th-gen/default.nix

```diff
@@ -0,0 +1,11 @@
+{ lib, pkgs, ... }:
+{
+  imports = [
+    ../.
+  ];
```

Comment:
Can you also reference this profile in the README as well as the flake?

Reply:
Done.